hack/import-restrictions.yaml

```diff
@@ -126,3 +126,24 @@
   - k8s.io/cli-runtime
   - k8s.io/client-go
   - k8s.io/sample-cli-plugin
+
+- baseImportPath: "./vendor/k8s.io/kube-controller-manager/"
+  allowedImports:
+  - k8s.io/apimachinery
+  - k8s.io/apiserver
+  - k8s.io/utils
+
+- baseImportPath: "./vendor/k8s.io/kube-proxy/"
+  allowedImports:
+  - k8s.io/apimachinery
+
+- baseImportPath: "./vendor/k8s.io/kube-scheduler/"
+  allowedImports:
+  - k8s.io/apimachinery
+  - k8s.io/apiserver
+  - k8s.io/utils
+
+- baseImportPath: "./vendor/k8s.io/kubelet/"
+  allowedImports:
+  - k8s.io/api
+  - k8s.io/apimachinery
```

Comment:
client-go

Reply:
nvmd. the restriction are ok as we only have the new config api types here.

Reply:
right, these are the only dependencies right now, will expand over time but it's good to track them here so we're aware when it happens